app/bundles/FormBundle/Translations/en_US/messages.ini

```diff
@@ -200,10 +200,14 @@ mautic.form.table.most.submitted="Most submitted forms"
 mautic.form.table.top.referrers="Top referrers"
 mautic.form.type.header="What type of form do you want to create?"
 mautic.form.type.campaign="Campaign Forms"
-mautic.form.type.campaign.description="Campaign forms can be selected to be the source of contacts when building a campaign. Some submit actions may not be available."
+mautic.form.type.one.differences="<li>Used to initiate automated campaign workflows</li>
+  <li>Limited form actions</li>
+  <li>Focuses on contact management</li>"
+mautic.form.type.two.differences="<li>Independent forms focused on flexibility</li>
+  <li>Contains all form actions</li>
+  <li>Actions for scoring, segmentation, and integration</li>"
```

Comment:
Let's move the HTML markup to the twig templates. This will cause troubles for translators. There should be 1 translation for each list item.